test/Controller/MiddlewareControllerTest.php

```diff
@@ -1,143 +0,0 @@
-<?php
-
-namespace LaminasTest\Mvc\Controller;
-
-use Laminas\EventManager\EventManager;
-use Laminas\EventManager\EventManagerInterface;
-use Laminas\Http\Request;
-use Laminas\Http\Response;
-use Laminas\Mvc\Controller\AbstractController;
-use Laminas\Mvc\Controller\MiddlewareController;
-use Laminas\Mvc\Exception\RuntimeException;
-use Laminas\Mvc\MvcEvent;
-use Laminas\Stdlib\DispatchableInterface;
-use Laminas\Stdlib\RequestInterface;
-use Laminas\Stratigility\MiddlewarePipe;
-use PHPUnit\Framework\TestCase;
-use Psr\Http\Message\ResponseInterface;
-
-/**
- * @covers \Laminas\Mvc\Controller\MiddlewareController
- *
- * Stratigility v2 does not support PHP 8
- * @requires PHP <8
```

Comment:
Are all these tests really not working on PHP 8? :| 

I feel like we just needed to enable them, rather than remove them.

Reply:
All test fail because of https://github.com/laminas/laminas-mvc/blob/3.4.x/src/MiddlewareListener.php#L138

Reply:
i have added `ignore_php_platform_requirements` for all php ( ;/) and `"laminas/laminas-stratigility": ">=2.0.1 <2.2",`